### ISSUE TITLE
Silicon Valley Series

### ISSUE THREAD

**Issue by PiciAkk** [stripedpajamas/hoku#9]
The prank named "ys" is really similar to what Gilfoyle did in the Silicon Valley series.  Did the idea come from here?
The series is great, and so is your program.

[Video Clip](https://www.youtube.com/watch?v=uS1KcjkWdoU)

**Comment by stripedpajamas** [comment]
Yeah, that was added by someone else but I believe you are correct about the inspiration 🤓
